src/ui/popup/main-page/style.less

```diff
@@ -4,6 +4,25 @@
     &__control {
         width: calc(100% - @size-border * 2);
     }
+
+    &__time {
+        background-color: @color-heading;
+        border-radius: 50%;
+        color: @color-control-fore;
+        cursor: pointer;
+        display: none;
+        height: @size-text-small-height;
+        pointer-events: none;
+        position: absolute;
+        right: 4.78125rem;
```

Comment:
Maybe try some beautiful number, like `calc(100% - @size-text-small-height / 2)`.

Reply:
I'm not quite sure if an beautiful number can come to this number as in the other original CSS for this icon the right was -0.2.....

Reply:
The original was placed in the corner, but here the whole width is `100% - 2 * @size-border`, the right edge of the 2nd button is `67% - @size-border / 3`, so maybe the number `67% - @size-border / 3 - 2 * @size-text-small-height / 3` should do the trick.

Reply:
Oh wait, it is `left` position, but for `right` try `33% + @size-border / 3`.